desktop/hardened-desktop/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hardened Desktop
+description: Overview of what Hardened Desktop is
+keywords: security, hardened desktop, enhanced container isolation, registry access management, admin controls, root access, administrators, docker desktop
+---
+>Note
+>
+>Hardened Desktop is available to Docker Business customers only.
+
+Hardened Desktop is a new security model for Docker Desktop. It's designed to provide administrators with a simple and powerful way to improve their organizations security posture for containerised development, without impacting the developer experience that Docker Desktop offers.
```

Comment:
sorry, annoying feedback here, but I think we should change 'administrators' to 'admins' throughout the docs. 

'Administrators' seems too formal. Unless we have a style guide or something which says that 'administrators' is preferred terminology and it's used elsewhere throughout the docs.